client/components/header/index.css

```
@@ -76,38 +86,47 @@
+
 .sticky-container {
   display: inline-flex;
   position: relative;
   width: 100%;
 }
+
 .sticky-msg {
   margin-left: 1.7rem;
   width: 100%;
   text-align: center;
   word-wrap: break-word;
```

Stylelint (17.3.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)